credentials/go.mod

```diff
@@ -2,6 +2,6 @@ module github.com/aws/aws-sdk-go-v2/credentials
 
 go 1.15
 
-require github.com/aws/aws-sdk-go-v2 v0.0.0-20200917190009-a19312abfd63
+require github.com/aws/aws-sdk-go-v2 v0.24.1-0.20200921191018-1a09374b6cac
```

Comment:
Curious why sometimes its adds the v0.24.1 prefix and sometimes doesn't

Reply:
`v0.24.1-0` is a made up version based on the last tagged replace in the repository that this commit is after. This is happening cause often I will use `go get` to update the hash versus using the `git log` command.